File: Java/Medium/WrongAnswer/count-lattice-points-inside-a-circle_8.java
class Solution {
    public int countLatticePoints(int[][] circles) {
        
        int count = 0;
        for(int i = 1; i < 101; i++){
            for(int j = 1; j < 101; j++){
                for(int[] circle : circles){
                    if((i - circle[0]) * (i - circle[0]) + (j - circle[1]) * (j - circle[1]) <= circle[2] * circle[2]){
                        count++;
                        break;
                    }
                }
            }
        }
        return count;
    }
}

File: C++/Medium/WrongAnswer/count-lattice-points-inside-a-circle_8.cpp
class Solution {
public:
    int countLatticePoints(vector<vector<int>>& circles) {
        
        int n = circles.size();
        int m = 100;
        vector<vector<int>> grid(m+1, vector<int>(m+1, 0));
        for (int i = 0; i < n; i++) {
            int x = circles[i][0];
            int y = circles[i][1];
            int r = circles[i][2];
            for (int j = x-r; j <= x+r; j++) {
                for (int k = y-r; k <= y+r; k++) {
                    if (j >= 0 && j <= m && k >= 0 && k <= m) {
                        if ((j-x)*(j-x) + (k-y)*(k-y) <= r*r) {
                            grid[j][k] = 1;
                        }
                    }
                }
            }
        }
        int res = 0;
        for (int i = 0; i <= m; i++) {
            for (int j = 0; j <= m; j++) {
                if (grid[i][j]) {
                    res++;
                }
            }
        }
        return res;
    }
};